client/src/components/Featured.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Image from './Image.jsx';
+import styles from './../styledComp/styles.jsx';
```

Comment:
I'm pretty sure you can use "../styledComp/styles.jsx" without the additional "./" Nitpicking, I know ;)